src/components/Questionnaire/QuestionnaireEditor.tsx

```
@@ -1552,70 +1711,126 @@ function QuestionEditor({
+                                      className="mr-2 size-4"
+                                    />
+                                    {t("delete")}
+                                  </DropdownMenuItem>
+                                </DropdownMenuContent>
+                              </DropdownMenu>
+                            </div>
                           </div>
                         </div>
-                      </div>
-                    ))}
+                      ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)